### DISCUSSION TITLE
Cinnamon on Alpine Linux, or make an fork from Alpine Linux with Cinnamon, lightdm? for lower memory usage, LM22 memory usage is over 1.3gb

### DISCUSSION THREAD

**Discussion by paulvonlux** [linuxmint/discussions#596] [Development] [answered]
Cinnamon on Alpine Linux, or make an fork from Alpine Linux with Cinnamon, lightdm? for lower memory usage, LM22 memory usage is over 1.3gb, that is not good, too much bloatware, MATE desktop use 400mb on Alpine Linux
![1](https://github.com/user-attachments/assets/1ef00d65-1b0e-4f8c-aa35-21c75de9dabc)

**Comment by paulvonlux** [comment]
Name it LMAE 1 (Linux Mint Alpine Edition) , that would be extraodinary

**Comment by paulvonlux** [comment]
package install and others with in Alpine are super fast, with doas addon also

**Comment by paulvonlux** [comment]
Also the space used by the Alpine Linux on disk is like 1.7-2gb max

**Comment by mtwebster** [comment, marked as answer]
Is Cinnamon not already available in Alpine linux? We don't package for other distributions, it would be up to someone else to maintain that.

As for an "LMAE", we're not interested in dealing with multiple package management systems at this time.

**Reply by paulvonlux** [reply]
Alpine Linux looks more efficient in resource consumption than Ubuntu or Debian, lowering resource consumption lowers the carbon foot print = lower global pollution, that's the one thing where MacOS excels and Linux don't

**Comment by Fantu** [comment]
From a fast search actually seems packaged only nemo (the file manager) and not cinnamon: https://pkgs.alpinelinux.org/package/edge/community/x86_64/nemo

**Comment by harshlalvani157-wq** [comment]
@paulvonlux 

Bro can you explain me about the aline cinnamon means it is in active development right ? And to have the cinnamon alpine we have build the combination of packages manully for now if how I'm beginner so please if possible explain in some easy words and I will use the proot on Android to run and x11 has display
